src/components/Main/LikeButton.js

```diff
@@ -0,0 +1,25 @@
+const LikeButton = () => {
```

Comment:
Buen trabajo Miguel aca no es necesario hacer el querySelector React se encarga de hacer el manejo del DOM, era lo que comentamos en clase, de esta forma en la que tu estas trabajando tu vista no cambia pero tu variable si agrega esto a tu codigo y observa tu consola, useState nos ayuda a tener informacion que cambie con el tiempo (Estado) y nos da un metodo para modificarla y que se renderice en nuestra vista, por tanto funciona tu codigo pero no se ve reflejado en tu vista te recomiendo hacer uso de useState
```suggestion
const LikeButton = () => {
  let count1 = 0;
  function like() {
    count1++;
    console.log(count1);
  }

  return (
    <div>
      <button onClick={like} id="b1">
        {count1}Likes
      </button>
      <button onClick={like} id="b2">
        Likes
      </button>
    </div>
  );
};

export default LikeButton;

```